docs/_data/default.yml

```diff
@@ -16,6 +16,9 @@ toc:
 
   - title: Install
     subfolderitems:
+      - page: VEBA (Knative)
```

Comment:
Is this just a placeholder or where is the doc (MD) for this link?

Reply:
This is for the doc (MD) stuff which is how a new section is created afaik. @pksrc can confirm this is correct but looking at the existing OF/EventBridge example, this is how you add a new section to page

Reply:
Ok, so we don't create the docs as part of this PR atomically then?

Reply:
I did but I totally forgot to include it in the commit. There's now `install-knative.md` which is the docs mimicking the existing two processors.